.travis.yml

```diff
@@ -4,8 +4,3 @@ services:
 language: python
 python:
 - 3.6
-script:
```

Comment:
This won't work due to needing a service token. Not sure how to handle travis for this repo